### DISCUSSION TITLE
Question: Can I compare LCCmax values across different particle-picking jobs?

### DISCUSSION THREAD

**Discussion by Neutrino0532** [SBC-Utrecht/pytom-match-pick#249] [Questions]
I recently encountered a situation where I picked two different particle types from the same tomogram and extracted particles using thresholds recommended by estimate_roc.py. However, I noticed some overlap between the two sets of picks.

Upon inspection, I observed that in the overlapping regions, one particle pick may have lower LCCmax values compared to the other. This observation suggests that I could potentially filter out false positives by removing overlapping particles with the worse LCCmax value.

However, I'm uncertain if comparing LCCmax values between different particle-picking jobs is valid, especially when the particles differ in size, shape, and symmetry. Does LCCmax remain a reliable metric under these conditions, or are there normalization or context-specific factors I need to consider?

**Comment by McHaillet** [comment]
Hey! Interesting question, I will move it to the Discussion page to save the thread.

It is possible, but the minimal requirement is that both searches use the same mask. The local normalization of the score is dependent on the size and shape of the mask and hence the cross correlation normalization changes for different masks. Besides that, as you mention as well,  size and shape will influence the LCCmax, so direct comparison can be tricky.

In case one of the particles is much larger than the other, those picks could be more reliable. So you could filter the reliable picks from the other list. 

If not the case, you could try to plot the scores of the two lists as 2 line plots (for example using [this code](https://sbc-utrecht.github.io/pytom-match-pick/Frequently-asked-questions-%28FAQ%29/#is-my-particle-handedness-correct)) to get some insight into their overlap. Or alternatively, make a 2D scatter plot of the the two lists at overlapping positions to see if there is any pattern.
